### ISSUE TITLE
It sometimes speeds up my game and has killed several runs because of it.

### ISSUE THREAD

**Issue by oteka** [networkMe/missinghud2#53]
I don't know how to explain it but whenever i have it open it speeds up the game sometimes like it will run at normal speed and then all of the sudden speeds up to like x2 times just like when you press tab in a gba emulator, it speeds up and i don't know why i already started it as administrator and that didn't work, closed and opened isaac again didn't work, restarted steam and nothing, i'm using MissingHUD2_v1.4.3 with afterbirth on the latest patch.

**Comment by networkMe** [comment]
I can't see how Missing HUD 2 can possibly speed up your game (or FPS) as, if anything, it should cause you to drop frames.

Do you experience this "speed-up" if you use something like OBS to record Afterbirth? It does exactly the same thing MHUD2 does.

**Comment by oteka** [comment]
No, it's only when MHUD is open i used bandicam to record the video but still it's not that i don't actually know how nor why but it's whenever i open MHUD that it happens, i tried even an older version and it's the same thing

**Comment by AutumnLeavesInTheWind** [comment]
It feels like the game is being so laggy that it's trying to fix the lag by speeding itself up?
Or some other weird Isaac black magic.
Maybe something to do with the framerate lock itself? I'm not sure.
I only had this problem when I accidentally had my PC in power saver mode, since then I've kept it on a custom high performance mode (100% power on everything at all times) and the game is fine now.
So it's either a performance issue on your part, asin you might just need a better PC, or you're like me and forget you have your PC set to run like crap. ;D

Sorry to post on an old post, I had this problem myself and have seemed to fix it so maybe it'll help anyone having this problem. ^^

**Comment by oteka** [comment]
I changed my settings to high performance like you said but i haven't used MHUD since then, i tried the item tracker for my stream and it actually did the same thing, sped up my game and still no fix, so i think it's not actually an MHUD problem but rather something with my PC? i really don't know i have a decent pc, FX 8320, R9 270X, 8 GB ram, i'll be tweaking some settings and try MHUD again to see if that fixes it, thank you anyways.

**Comment by AutumnLeavesInTheWind** [comment]
It's either your PC, or an Isaac problem. Your PC may not be powerful enough, and it feels like Isaac's way of dealing with it is unlocking the framerate.
It speeds up because the gameplay is tied to the framerate, I have frame count on for steam and times it speeds up it's going over 60fps to weird numbers like 84 or 120 (which sucks)
I tried MHUD on stream and it basically ran at 24fps through xsplit.
Other than that, on stream Isaac I get 55fps (which is kinda trash but I have GIFs and memes all over the screen and I use Xsplit don't judge XD)
Still, it feels like it's lagging our PC's because it's real-time injecting a DLL into the game. Because otherwise, I get 60fps through Isaac and no issues at all. I have 12GB of RAM, and I forget the name of my CPU but crossfire is disabled so it's again not using as much it could be.

But yes no it might be a PC issue. It feels that way. ^^;
